website/versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "v1.13.0",
   "v1.12.1",
   "1.12.0",
```

Comment:
Probably isn't related to this PR but should L4 and L8 have "v1.12.0"  and "v1.11.0" ?